purchases/src/main/kotlin/com/revenuecat/purchases/coroutinesExtensions.kt

```diff
@@ -10,13 +11,19 @@ import kotlin.coroutines.suspendCoroutine
  *
  * @warning This function is marked as [ExperimentalPreviewRevenueCatPurchasesAPI] and may change in the future.
  *
+ * @param fetchPolicy Specifies cache behavior for customer info retrieval (optional). Defaults to [CACHED_OR_FETCHED]
+ *
  * @throws [PurchasesException] with a [PurchasesError] if there's an error retrieving the customer info.
  * @return The [CustomerInfo] or a [PurchasesException] with the [PurchasesError]
  */
+@JvmOverloads
```

Comment:
Did we agree on leaving the door open to Java users? https://github.com/RevenueCat/purchases-android/pull/1012#discussion_r1223908647

Not sure if we'd like to mark this with `@JvmSynthetic` as we did for https://github.com/RevenueCat/purchases-android/pull/1066/files#r1229531480 https://github.com/RevenueCat/purchases-android/blob/333aba7d5a1dfdf09b6c702e05f39251b4ea651e/purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt#L100

If so, let me know and I'll replace `@JvmOverloads` by `@JvmSynthetic`.

Reply:
I think this API doesn't need to be available in Java, so I would maybe mark it with `@JvmSynthetic`. Thoughts @vegaro?